lib/tasks/avalon_ualbertalib.rake

```diff
@@ -66,6 +68,177 @@ namespace :avalon do
           exit 1
         end
       end
-      
+
+      desc "Return a list of collections: id's and labels"
+      task :list_collections => :environment do
+
+        Admin::Collection.all.to_a.collect.each do |collection|
+          puts "[%s] %s" % [collection.id, collection.name] 
+        end
+
+      end
+
+      desc "Given a single collection ID, return a list of collection members"
+      task :list_collection_members => :environment do
+
+        options = {}
+        option_parser = OptionParser.new do |opts|
+          opts.on("-c=ARG", "--collection_id=ARG", "collection id") { 
+            |id| options[:id] = id 
+          }
+        end
+        
+        args = option_parser.order!(ARGV) {}
+        option_parser.parse!(args)
+
+        if options[:id].nil?
+          puts "Manditory argument: -c id"
+          puts option_parser.help
+          exit 1
+        end
+        
+        obj = ActiveFedora::Base.find(options[:id])
+        if (obj.is_a?(Admin::Collection))
+          obj.media_objects.to_a.collect.each do |m| 
+            puts "%s" % [m.id] 
+          end;
+        else
+          puts "ERROR: not a Collection obj: [%s]" % [options[:id]] 
+        end
+
+      end
```

Comment:
Returns a list of ids representing all members of a specified collection
`bundle exec rake avalon:ualbertalib:list_collection_members -- --collection_id=mc87pq24j`